### DISCUSSION TITLE
Can I make plots for my friend

### DISCUSSION THREAD

**Discussion by JeffWeston** [Chia-Network/chia-blockchain#6025] [Q&A]
Please forgive me if this question has been answered, but I looked for it and can't seem to find it.

I have been plotting my chia for a week now and my friend wants to get involved, but he doesn't have a fast enough computer to plot. Is it possible for me to plot for him and then transfer the plots to his hard drive for his wallet to recognize, or will those plots link back to my wallet address?

If the above isn't possible, is there a way to alter the receive address for plots after I make them?

**Comment by tiny0little** [comment]
you need following details from your friend

```
$ chia plots create -h
Usage: chia plots create [OPTIONS]

Options:
  -c, --pool_contract_address TEXT
                                  Address of where the pool reward will be
                                  sent to. Only used if alt_fingerprint and
                                  pool public key are None

  -f, --farmer_public_key TEXT    Hex farmer public key
  -p, --pool_public_key TEXT      Hex public key of pool
```

your friend can get them by running ```chia keys show```

**Reply by JeffWeston** [reply]
Perfect. Where do I insert this info once received? does it go into each farm code, or is it inserted in the plotting process?

**Reply by tiny0little** [reply]
into command ```chia plots create```
